optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/pages/employeeroster/EmployeeRosterViewport.java

```diff
@@ -96,8 +95,7 @@ public void drawTicksAt(final IsElement target) {
         // Casting is preferable to avoid over-use of generics in the Viewport class
         final EmployeeLane employeeLane = (EmployeeLane) lane;
 
-        final EmployeeAvailability employeeAvailability = new EmployeeAvailability(tenantId, employeeLane.getEmployee(), GwtJavaTimeWorkaroundUtil.toLocalDate(start), start.toOffsetTime(), start.plusHours(8L)
-                .toOffsetTime());
+        final EmployeeAvailability employeeAvailability = new EmployeeAvailability(tenantId, employeeLane.getEmployee(), start, start.plusHours(8L));
```

Comment:
Why 8 hours?

Reply:
Default value, can change it to be full day of where it is clicked (when it fully working, you can drag the edge to change duration like in rotation)

Reply:
Needs to be a full day by default.
People think by default in days, as in "I can't work on the 5th of July because I am on vacation".
They only fine grain it to hours later on.